### DISCUSSION TITLE
Thank you - you saved us all

### DISCUSSION THREAD

**Discussion by zoulhh** [007revad/Video_Station_for_DSM_722#33] [General]
Thank you for bringing back the codecs that Synology deceitfully took away from us. You saved us all. (And for Video Station of course ).

All the best

Mick

**Comment by 007revad** [comment]
Your welcome

**Comment by snibchi** [comment]
Danke! Danke! Danke!

**Comment by Jaz123sun** [comment]
Thank you very much for bringing back VideoStation and its Codecs 😍
It took me a while to teach my whole family how to easily share video content with VideoStation ! And now it's works again !!!
Thank you very much for your art of scripting and sharing🤩
Best regards !
